src/import/importd.c

```diff
@@ -207,7 +208,7 @@ static void transfer_send_logs(Transfer *t, bool flush) {
                 _cleanup_free_ char *n = NULL;
                 char *e;
 
-                if (t->log_message_size >= sizeof(t->log_message))
+                if (t->log_message_size == sizeof(t->log_message))
```

Comment:
not sure this part really is an improvement... in the (impossible) case that the size is beyond the buffer you'll now process the buffer contents, which isn't that much better... in the old version we'd position e at the end of the buffer which is not the worst place...

Reply:
All right, I'll leave this commit out.